src/main/app/Resources/modules/security/registration/components/main.jsx

```diff
@@ -39,16 +55,52 @@ class RegistrationMain extends Component {
         title: 'Configuration',
         component: Optional
       }
-    ], this.props.facets.map(facet => ({
-      title: facet.title,
-      component: () => {
-        const currentFacet = <Facet facet={facet}/>
+    ], this.props.facets.map((facet) => ({
+      ...facet,
+      sections:
+        facet.sections !== undefined
+          ? [
+            ...facet.sections.map((section) => ({
```

Comment:
IMO, It would be better to put it in `Facet` component. We already loop on fields to modify them.